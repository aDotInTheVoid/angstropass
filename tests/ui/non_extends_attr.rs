#[angstropass::langs]
mod langs {
    #[something_else(with_attrs)]
    mod baz {}
}

fn main() {}
