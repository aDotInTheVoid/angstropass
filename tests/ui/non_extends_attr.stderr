error: expected #[extends]
 --> tests/ui/non_extends_attr.rs:3:7
  |
3 |     #[something_else(with_attrs)]
  |       ^^^^^^^^^^^^^^
